shell/platform/embedder/embedder.h

```diff
@@ -963,6 +963,43 @@ typedef const FlutterLocale* (*FlutterComputePlatformResolvedLocaleCallback)(
     const FlutterLocale** /* supported_locales*/,
     size_t /* Number of locales*/);
 
+/// Display refers to a graphics hardware system consisting of a framebuffer,
+/// typically a monitor or a screen. This ID is unique per display and is
+/// stable until the Flutter application restarts.
+typedef uint64_t FlutterDisplayId;
+
+typedef struct {
+  /// This size of this struct. Must be sizeof(FlutterDisplay).
+  size_t struct_size;
+
+  FlutterDisplayId display_id;
+
+  /// A double-precision floating-point value representing the actual refresh
+  /// period in seconds. This value may be zero if the device is not running or
+  /// unavaliable or unknown.
+  double refresh_rate;
+
+} FlutterDisplay;
```

Comment:
We should use `FlutterEngine` as the prefix for new types; I've already had to add a macro to solve one collision problem with macOS, and I'd rather we not introduce more potential collision points.

Reply:
Will do.